composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "firesphere/solr-search",
+  "name": "signify-nz/silverstripe-solr",
```

Comment:
Is this the name we want for this module? Just wondering if it would be better to match the new repo name